site/src/sphinx/server-decorator.rst

```diff
@@ -217,6 +217,37 @@ Therefore, you need to specify the decorators as extra parameters:
 A good real-world example of :api:`ServiceWithRoutes` is :api:`GrpcService`.
 See :ref:`server-grpc-decorator` for more information.
 
+Decorating a service by path mapping
+------------------------------------
+
+If you want to decorate a :api:`Service` by path mapping or router matching, you can specify a way to select
+the services which are being decorated with ``decorateUnder(pathPrefix, ...)`` or ``decorate(Route, ...)``.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.common.HttpHeaderNames;
+
+    UserService userService = ...;
+    ServerBuilder sb = new ServerBuilder();
+    sb.annotatedService("/users/{id}", userService)
```

Comment:
How about binding two services under `/users` (e.g. `/users/vip` and `/users/members`) to show that the `decorateUnder("/users")` decorates the both services?

Reply:
Added `vipService` and `memberService`.